### PR TITLE
[Examples Browser] playcanvas-observer 1.3.0

### PR DESCRIPTION
**Pull request by ellthompson**
Updates the playcanvas-observer dependency to version 1.3.0. Updates instances of the Observer class to match its type definition.

I confirm I have read the [contributing guidelines](https://github.com/playcanvas/engine/blob/master/.github/CONTRIBUTING.md) and signed the [Contributor License Agreement](https://docs.google.com/a/playcanvas.com/forms/d/1Ih69zQfJG-QDLIEpHr6CsaAs6fPORNOVnMv5nuo0cjk/viewform).

### CODE REVIEW

File: examples/src/app/example-iframe.tsx [hunk 1]
```diff
@@ -213,7 +213,7 @@ const ExampleIframe = (props: ExampleIframeProps) => {
         return false;
     };
 
-    const observer = new Observer({});
+    const observer = new Observer({}, {});
```
Comment (willeastcott) [inline examples/src/app/example-iframe.tsx:216]: Whoops, sorry but this is a mistake in the Observer JSDoc:

https://github.com/playcanvas/playcanvas-observer/blob/main/src/observer.js#L11

The options object should have square brackets around it to denote that it's optional.

Comment (willeastcott) [reply examples/src/app/example-iframe.tsx:216]: It's arguable that the first parameter should be marked as optional too, so you could be able to do:

```
const observer = new Observer();
```

@vkalpias what was the intention for the original API?

Comment (ellthompson) [reply examples/src/app/example-iframe.tsx:216]: Ahh yeah we should update the playcanvas-observer lib again in that case. I'll wait for @vkalpias answer.

Comment (vkalpias) [reply examples/src/app/example-iframe.tsx:216]: We should be able to make both arguments optional

Comment (willeastcott) [reply examples/src/app/example-iframe.tsx:216]: OK, I've attempted to fix this here: https://github.com/playcanvas/playcanvas-observer/pull/9

**Comment by ellthompson** [comment]
I've updated this with playcanvas-observer v1.3.1

**Review by willeastcott — APPROVED** [review]

**Pull request merged**